react-ui/src/pages/entry/EntryModal.jsx

```diff
@@ -160,6 +162,10 @@ function EntryModal() {
   const handleKeyUp = ({ key }) => {
     let indexChange = 0;
     if (key >= '0' && key <= '5') {
+      // ??? I would love it if we could track whether votes mobile or desktop
+      // ??? and if desktop, how many were keyed
```

Comment:
One option we could do for now is just create two columns on votes: is_desktop and is_keyed. Then we can create a variable in env.js for IS_DESKTOP where width > 800 (or something).

It's not full analytics, but I'd love to see 1. number of mobile voters and 2. adoption of the keyed vote so I can decide if it is worth designing a quick-vote system on mobile.

Lmk what you think.

Reply:
That's not a bad idea, gets an idea of how useful it is until we can implement proper analytics.

Reply:
Ok, I added this implementation. I followed MDN recommendations and code to check isTouchScreen instead of width. Works well, but can't test on mobile.

Reply:
Neat, that'll probably pick some tablet-like devices like Chromebooks that also have touchscreens, but I don't think we get a lot of usage on those devices. I think this approach will be a reasonable estimate.

We can test once we merge this PR and it's available in staging.